Integrations/integration-Threat_Grid.yml

```diff
@@ -785,6 +785,13 @@ script:
       description: For malicious files, the vendor that made the decision
     - contextPath: File.Malicious.Description
       description: For malicious files, the reason for the vendor to make the decision
+    important:
+    - contextPath: File.Malicious.Vendor
```

Comment:
I think it should be:
File(val.Malicious)
See other integrations as examples.
And then you should remove the second important entry